app/webhook.go

```diff
@@ -388,10 +388,10 @@ func (a *App) GetIncomingWebhooksForTeamPage(teamId string, page, perPage int) (
 		return nil, model.NewAppError("GetIncomingWebhooksForTeamPage", "api.incoming_webhook.disabled.app_error", nil, "", http.StatusNotImplemented)
 	}
 
-	if result := <-a.Srv.Store.Webhook().GetIncomingByTeam(teamId, page*perPage, perPage); result.Err != nil {
-		return nil, result.Err
+	if hooks, err := a.Srv.Store.Webhook().GetIncomingByTeam(teamId, page*perPage, perPage); err != nil {
```

Comment:
We here prefer the approach of transforming it to a more "idiomatic" way to thandle the errors, something like:

```go
hooks, err := a.Srv.Store.Webhook().GetIncomingByTeam(teamId, page*perPage, perPage)
if err != nil {
    return nil, err
}
return hooks, nil
```

Reply:
probably makes sense somethin like `return a.Srv.Store.Webhook().GetIncomingByTeam(teamId, page*perPage, perPage)`